wire-ios-sync-engine/Source/Synchronization/MLSGroupRepairAgent.swift

```diff
@@ -39,7 +38,7 @@ final class MLSGroupRepairAgent: MLSGroupRepairAgentProtocol {
     }
 
     private let syncStateSubject: CurrentValueSubject<SyncState, Never>
-    var syncStatePublisher: AnyPublisher<SyncState, Never> {
+    private var syncStatePublisher: AnyPublisher<SyncState, Never> {
```

Comment:
suggestion: better to inject the `AnyPublisher` rather than the current value subject since we only want to receive events and not send them.

Reply:
done